src/components/NewCommentForm.tsx

```diff
@@ -75,25 +163,43 @@ export const NewCommentForm: React.FC = () => {
             id="comment-body"
             name="body"
             placeholder="Type comment here"
-            className="textarea is-danger"
+            className={classNames('textarea', {
+              'is-danger': bodyCommentError,
+            })}
+            value={bodyComment}
+            onChange={e => {
+              setBodyComment(e.target.value);
+              setBodyCommentError(false);
+            }}
           />
         </div>
 
-        <p className="help is-danger" data-cy="ErrorMessage">
-          Enter some text
-        </p>
+        {bodyCommentError && (
+          <p className="help is-danger" data-cy="ErrorMessage">
+            Enter some text
+          </p>
+        )}
       </div>
 
       <div className="field is-grouped">
         <div className="control">
-          <button type="submit" className="button is-link is-loading">
+          <button
+            type="submit"
+            className={classNames('button is-link', {
+              'is-loading': buttonAddLoading,
+            })}
+          >
             Add
           </button>
         </div>
 
         <div className="control">
           {/* eslint-disable-next-line react/button-has-type */}
-          <button type="reset" className="button is-link is-light">
+          <button
```

Comment:
You have an eslint disable comment here. It's better to fix the issue instead of disabling the linter. In this case, you should specify the button type as 'button' instead of 'reset' to avoid unexpected form reset behavior.